### ISSUE TITLE
Developer tools throwing "Uncaught (in promise) Error: Cannot match any routes. URL Segment: 'ombi' - Apache Reverse Proxy

### ISSUE THREAD

**Issue by gazsux** [tidusjar/Ombi#3084]
**Describe the bug**
I'm trying to use Apache as a reverse proxy to reach Ombi via http://servername.domain/ombi (internal only, no external access). When loading the URL in incognito mode, developer tools' console throws the following error:

>ERROR Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'ombi'
Error: Cannot match any routes. URL Segment: 'ombi'       core.js:14597 
    at t.noMatchError (router.js:2469)
    at e.selector (router.js:2450)
    at e.error (catchError.js:34)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (tap.js:61)
    at I (zone.js:814)
    at I (zone.js:771)
    at zone.js:873
    at t.invokeTask (zone.js:421)
    at Object.onInvokeTask (core.js:16147)
    at t.invokeTask (zone.js:420)
    at e.runTask (zone.js:188)
    at g (zone.js:595)
Sd @ core.js:14597
t.handleError @ core.js:14645
next @ core.js:16628
i @ core.js:12609
e.__tryOrUnsub @ Subscriber.js:196
e.next @ Subscriber.js:134
e._next @ Subscriber.js:77
e.next @ Subscriber.js:54
e.next @ Subject.js:47
e.emit @ core.js:12593
(anonymous) @ core.js:16178
t.invoke @ zone.js:388
e.run @ zone.js:138
t.runOutsideAngular @ core.js:16115
onHandleError @ core.js:16178
t.handleError @ zone.js:392
e.runGuarded @ zone.js:154
t @ zone.js:677
n.microtaskDrainDone @ zone.js:686
g @ zone.js:602
Promise.then (async)
h @ zone.js:578
t.scheduleTask @ zone.js:410
onScheduleTask @ zone.js:297
t.scheduleTask @ zone.js:401
e.scheduleTask @ zone.js:232
e.scheduleMicroTask @ zone.js:252
j @ zone.js:862
t.then @ zone.js:962
t.appInitializer @ router.js:5533
t.runInitializers @ core.js:15612
(anonymous) @ core.js:16631
(anonymous) @ core.js:16724
(anonymous) @ core.js:16629
t.invoke @ zone.js:388
onInvoke @ core.js:16156
t.invoke @ zone.js:387
e.run @ zone.js:138
t.run @ core.js:16070
t.bootstrapModuleFactory @ core.js:16620
(anonymous) @ main.ts:39
o @ bootstrap:63
(anonymous) @ bootstrap:195
(anonymous) @ bootstrap:195

> Uncaught (in promise) Error: Cannot match any routes. URL Segment: 'ombi'      router.js:2469 
    at t.noMatchError (router.js:2469)
    at e.selector (router.js:2450)
    at e.error (catchError.js:34)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (Subscriber.js:80)
    at e.error (Subscriber.js:60)
    at e._error (tap.js:61)

**Expected behavior**
My goal is to be able to navigate to http://servername.domain/ombi to reach Ombi instead of http://servername.domain:5000

**Screenshots**
Errors in Developer Tools: https://prnt.sc/oi4gvr
Reverse proxy rules in httpd-vhosts.conf: https://prnt.sc/oi4hqh

**Logs (Logs directory where Ombi is located)**
No applicable log entries related to the reverse proxy issue

**Desktop (please complete the following information):**
 - OS: Windows Server 2012R2

**Ombi Version (please complete the following information):**
 - Version: 3.0.4680
- Media Server: Plex

**Additional context**

**Comment by tidusjar** [comment]
Did you set the base URL in the settings?

**Comment by gazsux** [comment]
> Did you set the base URL in the settings?

Yes, the base URL has been set as "http://servername.domain/ombi" and I restarted Ombi by running "net stop ombi" and "net start ombi" in CMD on the server.

**Comment by tidusjar** [comment]
It's needs to be set as /ombi

**Comment by gazsux** [comment]
> 
> 
> It's needs to be set as /ombi

Every time I try and set it as "/ombi" there's an error in an orange pop-up that says "The URL "/ombi" is not valid. Please format it correctly e.g. http://www.google.com"

**Comment by tidusjar** [comment]
That's your application URL. You have that set correctly then. 

You need to set the Base URL in the Ombi settings

**Comment by gazsux** [comment]
> That's your application URL. You have that set correctly then.
> You need to set the Base URL in the Ombi settings
Gotcha, where do I set the base url?

**Comment by gazsux** [comment]
Nvm, got it. I had trouble finding specific instructions so if anyone else needs to set Ombi's URL on Windows you do it from command prompt. I had installed Ombi as a service, so I stopped the Ombi service, navigated to C:\Program Files (x86)\Ombi\ (that's where I installed Ombi) and typed "ombi.exe --baseurl /ombi". This started the Ombi server and set my new Ombi URL. I then stopped the server in CMD and restarted the service. All is well. Thank you @tidusjar
